stellargraph/layer/graph_attention.py

```diff
@@ -208,38 +232,37 @@ def build(self, input_shape):
             self.attn_kernels.append([attn_kernel_self, attn_kernel_neighs])
         self.built = True
 
-    def call(self, inputs, **kwargs):
+    def call(self, inputs):
         """
-        Applies the layer.
+        Creates the layer as a Keras graph
 
-        Args:
-            inputs (list): list of inputs with 2 items: node features (matrix of size N x F),
-                and graph adjacency matrix (size N x N), where N is the number of nodes in the graph,
-                F is the dimensionality of node features
+        Notes:
+            This does not add self loops to the adjacency matrix.
+            The output indices are only used when `final_layer=True`
 
+        Args:
+            inputs (list): list of inputs with 3 items:
+            node features (size b x N x F),
+            output indices (size b x M),
+            graph adjacency matrix (size b x N x N),
+            where N is the number of nodes in the graph,
```

Comment:
It might be clearer to say what b represents here.

Reply:
Good point, I'll add a comment.